clvm-traits/docs/derive_macros.md

```diff
@@ -24,7 +24,7 @@ For example:
 - `(A, B, C)` is encoded as `(A . (B . C))`, since every cons-pair must contain two values.
 - `(A, B, C, D)` is encoded as `(A . (B . (C . D)))` for the same reason as above.
 
-```rust
+```rust compile_fail
```

Comment:
Why does this need to be compile_fail?

Reply:
because it's assuming that `Allocator` and `NodePtr` implements `FromClvm` and `ToClvm`. Once all of this is sorted, I expect to restore this.

doctest tries to build it, and it fails because it's using this local version of clvm-traits